.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing
+This page contains the contributing guidelines.
+
+## Setup
+### prerequisite
+- Node.js installed. 
+ _Note: Node 6 or greater would be better for "best results"._
+- Package manager - [yarn](https://yarnpkg.com/en/docs/getting-started) or [npm](https://docs.npmjs.com/)
+
+
+
+### Project Setup in your local machine
+-   Fork the **just** repo at [https://github.com/microsoft/just](https://github.com/microsoft/just).
+-   `git clone <your-clone-url> && cd just`
+
+### Dependency Installation
+
+- Using npm
+```bash
+$ npm install
+$ npm run build
+$ npm link
+```
+
+
+- Using yarn
+```bash
+$ yarn  
+$ yarn build
+$ yarn link
```

Comment:
Don't think we need yarn link here. Maybe inside your for local testing?

Reply:
actually yup local testing only I kept that. Should I keep it ?

Reply:
You should remove it for now. I'm not sure how to best to word the local testing yet :) I've been using lots of yarn linking manually, etc. so far.